spec/lib/reporting/identity_verification_report_spec.rb

```diff
@@ -16,16 +16,19 @@
       fetch: [
         # Online verification user
         { 'user_id' => 'user1', 'name' => 'IdV: doc auth welcome visited' },
+        { 'user_id' => 'user1', 'name' => 'IdV: doc auth welcome submitted' },
         { 'user_id' => 'user1', 'name' => 'IdV: doc auth image upload vendor submitted' },
         { 'user_id' => 'user1', 'name' => 'IdV: final resolution', 'identity_verified' => '1' },
 
         # Letter requested user (incomplete)
         { 'user_id' => 'user2', 'name' => 'IdV: doc auth welcome visited' },
+        { 'user_id' => 'user2', 'name' => 'IdV: doc auth welcome submitted' },
```

Comment:
These only need to be changed if we're including them in `to_csv` which is debatable.